dockers/docker-sonic-mgmt/Dockerfile.j2

```diff
@@ -60,6 +60,7 @@ RUN pip install cffi==1.10.0 \
                 pysnmp==4.2.5 \
                 pytest-repeat \
                 pytest-html \
+                allure-pytest==2.8.22 \
```

Comment:
why not use the latest version?

Reply:
As sonic-mgmt framework is on python2, I choose the latest allure-pytest version that supports python2, another allure-pytest require higher version than python2.7